app/models/throttle.rb

```diff
@@ -71,15 +71,15 @@ class Throttle < ApplicationRecord
   # @return [Throttle]
   def self.for(throttle_type:, user: nil, target: nil)
     throttle = if user
-      find_or_create_by(user: user, throttle_type: throttle_type)
-    elsif target
-      if !target.is_a?(String)
-        raise ArgumentError, "target must be a string, but got #{target.class}"
-      end
-      find_or_create_by(target: target, throttle_type: throttle_type)
-    else
-      raise 'Throttle must have a user or a target, but neither were provided'
-    end
```

Comment:
what if we tried this style so we can indent less overall? Could undo some of the ternary conversions if needed
```ruby
    throttle =
      if user
        find_or_create_by(user: user, throttle_type: throttle_type)
      elsif target
        if !target.is_a?(String)
          raise ArgumentError, "target must be a string, but got #{target.class}"
        end
        find_or_create_by(target: target, throttle_type: throttle_type)
      else
        raise 'Throttle must have a user or a target, but neither were provided'
      end
```

Reply:
Yeah I like that better. Will update.

Reply:
Updated in 22e60e3eb.

Reply:
Thanks! I just realized we can probably combine this with 

```
MultilineAssignmentLayout
  EnforcedStyle: new_line
```

to help make it more consistent?

https://www.rubydoc.info/gems/rubocop/0.41.1/RuboCop/Cop/Style/MultilineAssignmentLayout

Reply:
I could go either way as far as leaving this to developer preference. That being said, as it stands, the configuration doesn't enforce consistency, so something like the following is technically valid 🤮 

```rb
foo = if expression
        'bar'
else
  'baz'
end
```

Maybe worth enforcing just to avoid that.